README.md

```diff
@@ -56,6 +57,7 @@ Consult the [Icinga DB configuration documentation] on what options there are.
 docker run -d \
 	--network icinga \
 	--restart always \
+        --name icingadb \
 	-e ICINGADB_REDIS_HOST=redis-icingadb \
```

Comment:
It's still mixed indentation here.

Reply:
It should be updated as of commit 1bbcc02
```diff
 	--restart always \
-        --name icingadb \
+	--name icingadb \
 	-e ICINGADB_REDIS_HOST=redis-icingadb \
```

Reply:
Ok, I see what you mean.
I just pushed another fix.